### PR TITLE
Mod/11291 update error message

### PR DESCRIPTION
**Pull request by jagu65**
**High level description:**
For the Autocomplete component, the 'minimum characters' error was removed and the underline error message for 'No results found' was also removed.

**Technical details:**
A new prop was added to Autocomplete to check if a minimum character error was desired. This was done to maintain this functionality to preserve this functionality for future use.

**JIRA Ticket:**
[DEV-11291](https://federal-spending-transparency.atlassian.net/browse/DEV-11291)

**Mockup:**
https://app.zeplin.io/project/61f8076f4ef77f11fcc37f55

The following are ALL required for the PR to be merged:

Author:
- [x] Linked to this PR in JIRA ticket
- [ ] Scheduled demo including Design/Testing/Front-end OR Provided instructions for testing in JIRA and PR `if applicable`
- [x] Verified cross-browser compatibility: Chrome, Safari, Firefox, Edge
- [x] Verified mobile/tablet/desktop/monitor responsiveness
- [ ] Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)
- [ ] Added Unit Tests for helper functions, reducers, models and Container/Component Interactivity Expectations `if applicable` [React Testing Library](react-testing-library.md)
- [ ] [API contract](https://github.com/fedspendingtransparency/usaspending-api/tree/dev/usaspending_api/api_contracts) updated `if applicable`
- [ ] [Component Library Integration Status Report](https://github.com/fedspendingtransparency/data-act-documentation/blob/data-transparency-ui/frontend_apps/component-library-integration-status.md) updated `if applicable`

Reviewer(s):
- [ ] Design review complete `if applicable`
- [ ] [API #1234](https://github.com/fedspendingtransparency/usaspending-api/pull/1234) merged concurrently `if applicable`
- [ ] Code review complete

### CODE REVIEW

**Comment by nick-torres** [comment]
assigned myself this one bc my work actually depends on this

File: src/js/containers/search/filters/AgencyListContainer.jsx [hunk 1]
```diff
@@ -43,7 +43,8 @@ export default class AgencyListContainer extends React.Component {
         let noResults = false;
 
         // Format results of search for use in Autocomplete component
-        if (results && results.length > 0) {
+        if (results) {
+            console.log('results: ', results);
```
Comment (anjenkin) [inline src/js/containers/search/filters/AgencyListContainer.jsx:47]: console log to remove

**Review by anjenkin — CHANGES_REQUESTED** [review]
looks good.  just need to remove a console.log

File: src/js/containers/search/filters/AgencyListContainer.jsx (hunk 1 above)
Comment (jagu65) [reply src/js/containers/search/filters/AgencyListContainer.jsx]: Dang console logs, they're so sneaky. I've removed it.

**Review by anjenkin — APPROVED** [review]

**Pull request merged**